mdal/frmts/mdal_selafin.cpp

```diff
@@ -615,7 +615,7 @@ void MDAL::SelafinFile::ignoreArrayLength( )
 MDAL::DriverSelafin::DriverSelafin():
   Driver( "SELAFIN",
           "Selafin File",
-          "*.slf",
+          "*.slf;;*.ser;;*.geo;;*.res",
```

Comment:
I understand *.ser, but does "geo" and "res" file has exactly the same internal file structure as slf file? If not, than the driver will not be able to open the files

Reply:
@PeterPetrik Yes. The internal structure is exactly the same.

It's the header inside the binary file that determine if the file is a SELAFIN file, not the extension.